torch/_tensor_str.py

```diff
@@ -284,6 +288,10 @@ def _str_intern(inp):
     # within this function.
     # TODO(albanD) This needs to be updated when more than one level is supported
     self, tangent = torch.autograd.forward_ad.unpack_dual(inp)
+    
+    # print shape of the tensor
+    if PRINT_OPTS.show_shape:
+        suffixes.append('shape= ' + str(tuple(self.shape)))
```

Comment:
remove space in 'shape= '

Reply:
pushed this [here](https://github.com/pytorch/pytorch/pull/64136/commits/65ab1ff26cb9b488498b5d19434c2d03f6f419fa)